ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/RangeActionFilter.java

```diff
@@ -248,22 +252,47 @@ private void removeRangeActionsWithBiggestImpact(Set<RangeAction> rangeActions,
      * it will be considered greater.
      * If both RAs have the same priority, then absolute sensitivities will be compared.
      */
-    private int comparePrioritiesAndSensitivities(RangeAction ra1, RangeAction ra2, FlowCnec
-            cnec, SensitivityResult sensitivityResult) {
+    private int comparePrioritiesAndSensitivities(RangeAction ra1, RangeAction ra2, FlowCnec cnec, OptimizationResult optimizationResult) {
```

Comment:
rename to comparePrioritiesAndPotentialGain

Reply:
ok